xml/metering-neutron_meters.xml

```diff
@@ -26,134 +26,6 @@
-     <entry>Update requests for this network</entry>
-    </row>
-    <row>
-     <entry>network.delete</entry>
-     <entry>Delta</entry>
-     <entry>Network</entry>
-     <entry>Network ID</entry>
-     <entry>Notification</entry>
-     <entry>Delete requests for this network</entry>
-    </row>
-    <row>
-     <entry>subnet</entry>
-     <entry>Gauge</entry>
-     <entry>Subnet</entry>
-     <entry>Subnet ID</entry>
-     <entry>Notification</entry>
-     <entry>Existence of subnet</entry>
-    </row>
-    <row>
-     <entry>subnet.create</entry>
-     <entry>Delta</entry>
-     <entry>Subnet</entry>
-     <entry>Subnet ID</entry>
-     <entry>Notification</entry>
-     <entry>Creation requests for this subnet</entry>
-    </row>
-    <row>
-     <entry>subnet.update</entry>
-     <entry>Delta</entry>
-     <entry>Subnet</entry>
-     <entry>Subnet ID</entry>
-     <entry>Notification</entry>
-     <entry>Update requests for this subnet</entry>
-    </row>
-    <row>
-     <entry>subnet.delete</entry>
-     <entry>Delta</entry>
-     <entry>Subnet</entry>
-     <entry>Subnet ID</entry>
-     <entry>Notification</entry>
-     <entry>Delete requests for this subnet</entry>
-    </row>
-    <row>
-     <entry>port</entry>
-     <entry>Gauge</entry>
-     <entry>Port</entry>
-     <entry>Port ID</entry>
-     <entry>Notification</entry>
-     <entry>Existence of port</entry>
-    </row>
-    <row>
-     <entry>port.create</entry>
-     <entry>Delta</entry>
-     <entry>Port</entry>
-     <entry>Port ID</entry>
-     <entry>Notification</entry>
-     <entry>Creation requests for this port</entry>
-    </row>
-    <row>
-     <entry>port.delete</entry>
-     <entry>Delta</entry>
-     <entry>Port</entry>
-     <entry>Port ID</entry>
-     <entry>Notification</entry>
-     <entry>Delete requests for this port</entry>
-    </row>
-    <row>
-     <entry>router</entry>
-     <entry>Gauage</entry>
-     <entry>Router</entry>
-     <entry>Router ID</entry>
-     <entry>Notification</entry>
-     <entry>Existence of router</entry>
-    </row>
-    <row>
-     <entry>router.create</entry>
-     <entry>Delta</entry>
-     <entry>Router</entry>
-     <entry>Router ID</entry>
-     <entry>Notification</entry>
-     <entry>Creation requests for this router</entry>
-    </row>
-    <row>
-     <entry>router.update</entry>
-     <entry>Delta</entry>
-     <entry>Router</entry>
-     <entry>Router ID</entry>
-     <entry>Notification</entry>
-     <entry>Update requests for this router</entry>
-    </row>
-    <row>
-     <entry>router.delete</entry>
-     <entry>Delta</entry>
-     <entry>Router</entry>
-     <entry>Router ID</entry>
-     <entry>Notification</entry>
-     <entry>Delete requests for this router</entry>
-    </row>
-    <row>
-     <entry>ip.floating</entry>
-     <entry>Gauge</entry>
-     <entry>Ip</entry>
-     <entry>IP ID</entry>
-     <entry>Notification, Pollster</entry>
-     <entry>Existence of floating ip</entry>
-    </row>
```

Comment:
It looks like all the content of the tbody has been removed, so I think the tbody tags also need to be removed to avoid some build errors.

Reply:
Is this really what you wanted? It looks strange in the actual document
![operations-ceilometer_screenshot_20180614_1835](https://user-images.githubusercontent.com/33355939/41446006-fd7175a6-7001-11e8-9aea-05f8cee0a090.png)
This PR fails Travis checks. Validation works with this content...adding lines for </row and </entry>.
 <tbody>
    <row>
     <entry></entry>
     <entry></entry>
     <entry></entry>
     <entry></entry>
     <entry></entry>
     <entry></entry>
    </row>
   </tbody>

Reply:
This content makes validation successful.
   <tbody>
    <row>
     <entry></entry>
     <entry></entry>
     <entry></entry>
     <entry></entry>
     <entry></entry>
     <entry></entry>
    </row>
   </tbody>

Reply:
Let me just remove the Networking meters section. All these networking meters were removed as of Pike release upstream